.github/workflows/coverage.yaml

```diff
@@ -16,8 +16,43 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v3
 
-      - name: Fetch target branch
-        run: git fetch origin ${{ github.base_ref }}:refs/remotes/origin/${{ github.base_ref }}
+      - name: Fetch github branches and detect introduces .py files
```

Comment:
I think this action should simplify a lot of this logic: https://github.com/marketplace/actions/changed-files

It seems like you can do:
```yaml
- name: Get all changed python test files
  id: changed-python-files
  uses: tj-actions/changed-files@v45
  with:
    # Avoid using single or double quotes for multiline patterns
    files: |
       tests/**.py
```

Reply:
Should check it out. To see how it works on  removed/deleted files

Reply:
hm, does not work. as I remember this format: 
  `${{ steps.changed-python-files.outputs.all_changed_files }}`
is not supported in github actions anymore. the script has to write to environment directly

Reply:
Hi @winsvega, I had a look at changed-files, see #839. It looks pretty useful.